doc/releases/changelog-dev.md

```diff
@@ -672,6 +670,13 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
 
 <h4>Other improvements</h4>
 
+* `qml.draw_mpl` can now split deep circuits over multiple figures via a `max_length` keyword argument.
```

Comment:
Oh, I thought this PR indeed belongs to "Better drawing functionality"?

Reply:
🤷 Too much going on in the changelog to find a spot.